runtime/src/main/java/org/corfudb/runtime/object/transactions/OptimisticTransactionalContext.java

```diff
@@ -56,6 +49,13 @@
     private final Set<ICorfuSMRProxyInternal> modifiedProxies =
             new HashSet<>();
 
+    /** Keep the version (at global snapshot time)
```

Comment:
I also didn't see a huge improvement in the multithreaded case. It improves significantly in memory runs and disk singlethreaded. (the runs that are not shown currently in the performance graphs.) We might plateau in the multithreaded disk runs because of another bottleneck. I will try to investigate further what is happening.